listenbrainz/webserver/views/api.py

```diff
@@ -177,6 +177,36 @@ def get_playing_now(user_name):
     })
 
 
+@api_bp.route("user/<user_name>/stats/artist")
```

Comment:
Can we move this endpoint to a different file and a different blueprint? We'll probably have lots of these endpoints soon.